CMakeLists.txt

```diff
@@ -946,6 +945,26 @@ else()
     message(FATAL_ERROR "${CMAKE_CXX_COMPILER_ID} not yet supported.")
 endif()
 
+if(NOT MSVC)
```

Comment:
Change to `if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")`. If I recall correctly MSVC will be set when using ClangCL because it supports the same command line options.